MdsSupportingClasses/ShippingPackageData.php

```diff
@@ -131,6 +145,7 @@ public function build($packages, $input)
      */
     public function extractRequiredFields(array $array, array $packages)
     {
+        //$array = $array[0];
```

Comment:
Please remove comments

Reply:
Apologies, I'll remove that and anything else I find.

Reply:
This has been removed.